### PR TITLE
[4] Layout One time emergency passwords correctly

### PR DESCRIPTION
**Pull request by PhilETaylor**
### Summary of Changes

Correct HTML so that the "One time emergency passwords" can be laid out and copy and pasted to a safe place with ease.

### Testing Instructions

Enable 2FA as a user, view your "One time emergency passwords". 

### Actual result BEFORE applying this Pull Request

Screenshot 2021-08-20 at 16 25 24

### Expected result AFTER applying this Pull Request
Screenshot 2021-08-20 at 16 23 14

### CODE REVIEW

File: administrator/components/com_users/tmpl/user/edit.php [hunk 1]
```diff
@@ -103,9 +103,7 @@
 				</div>
 			<?php else : ?>
 				<?php foreach ($this->otpConfig->otep as $otep) : ?>
-					<span class="col-lg-3">
-						<?php echo substr($otep, 0, 4); ?>-<?php echo substr($otep, 4, 4); ?>-<?php echo substr($otep, 8, 4); ?>-<?php echo substr($otep, 12, 4); ?>
-					</span>
+					<?php echo substr($otep, 0, 4); ?>-<?php echo substr($otep, 4, 4); ?>-<?php echo substr($otep, 8, 4); ?>-<?php echo substr($otep, 12, 4); ?><br>
```
Comment (bembelimen) [inline administrator/components/com_users/tmpl/user/edit.php]:
```suggestion
					<?php echo wordwrap($otep, 4, '-', true); ?><br>
```

Just an idea...

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on efaa8e2424270508189d519ff98798971cf5de61<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35254">issues.joomla.org/tracker/joomla-cms/35254</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on efaa8e2424270508189d519ff98798971cf5de61<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35254">issues.joomla.org/tracker/joomla-cms/35254</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35254">issues.joomla.org/tracker/joomla-cms/35254</a>.</sub>

**Pull request merged**